examples/utf8_benchmark_client.rs

```diff
@@ -108,9 +108,9 @@ fn main() -> io::Result<()> {
                 panic!("should never happen");
             }
 
-            let last_byte_read = unsafe { buf.get_unchecked(n - 1) };
+            let last_byte_read = buf[n - 1];
 
-            if *last_byte_read == DELIMITER as u8 {
```

Comment:
During the rebase I noticed this one too. The optimizer can't probably prove `n - 1` is in bounds, but I'd still go for this because it removes the `unsafe` in example code.

Reply:
This isn't supposed to be an example, it needs to be moved somewhere else. We should move this to a `bench` subdirectory and use criterion for this.

Reply:
Restored